finder-app/finder.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+filesdir=$1
+searchstr=$2
+
+if [ $# != 2 ]
+then
+echo "Number of paramters are incorrect"
```

Comment:
@Rajs060699 how could you make this error message more descriptive?  Also for readability typically you'd indent if/then similar to what you'd do for C code.